src/store/index.js

```diff
@@ -0,0 +1,16 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+import PrincipalStore from './modules/principal';
+import ServicesStore from './modules/services';
+import UatStore from './modules/uat';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  modules: {
```

Comment:
I believe this has some parameter strict : true , so that you can't change Store directly